### PR TITLE
Fix collision bug and setup tests for GameLabP5

### PR DESCRIPTION
**Pull request by caleybrock**

### CODE REVIEW

File: apps/src/gamelab/GameLabP5.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ GameLabP5.prototype.init = function (options) {
     }
 
     if (sprite.collider instanceof this.CircleCollider) {
-      return window.p5.dist(mousePosition.x, mousePosition.y, sprite.collider.center.x, sprite.collider.center.y) < sprite.collider.radius;
+      return this.dist(mousePosition.x, mousePosition.y, sprite.collider.center.x, sprite.collider.center.y) < sprite.collider.radius;
```
Comment (caleybrock) [inline apps/src/gamelab/GameLabP5.js:205]: This is the actual fix of the collision bug.

File: apps/Gruntfile.js [hunk 2]
```diff
@@ -318,6 +318,8 @@ module.exports = function (grunt) {
         {pattern: 'lib/**/*', watched: false, included: false, nocache: true},
         {pattern: 'build/**/*', watched: false, included: false, nocache: true},
         {pattern: 'static/**/*', watched: false, included: false, nocache: true},
+        {pattern: 'lib/p5play/p5.js'},
+        {pattern: 'lib/p5play/p5.play.js'},
```
Comment (islemaster) [inline apps/Gruntfile.js]: @pcardune I'm pretty sure we did this correctly, can you confirm?  p5.js and p5.play.js absolutely insist on being included as script tags so they can load stuff onto window (they are not requirejs-compatible) so we put them here to include them in tests ahead of any of the actual test files.

It's not great that this means they'll be included for any/every test run of any kind, but I didn't see a better way to do this and use them in our gamelab unit tests.  I also doubt they'll slow the build down very much, as they're both prebuilt libraries, leaf nodes in the dependency tree.

Thoughts?

Comment (pcardune) [inline apps/Gruntfile.js]: You could import them where you need them using the webpack script-loader. See https://github.com/webpack/script-loader.

Comment (caleybrock) [inline apps/Gruntfile.js]: updated - thanks for the suggestion @pcardune

**Comment by balderdash** [comment]
LGTM

**Pull request merged**